tests/ppx/react/expected/lowercases.res.txt

```diff
@@ -0,0 +1,9 @@
+@@jsxConfig({version: 4, mode: "classic"})
+
+let _ = ReactDOM.createDOMElementVariadic("div", [])
+let _ = ReactDOM.createDOMElementVariadic("div", ~props={key: "k"}, [])
```

Comment:
How can the same function`createDOMElementVariadic` accept these types?

Reply:
Not sure I understand your question. The second labeled argument is optional.
```rescript
@variadic @module("react")
external createDOMElementVariadic: (
  string,
  ~props: domProps=?,
  array<React.element>,
) => React.element = "createElement"
```

Reply:
It reminds me of this https://forum.rescript-lang.org/t/ann-more-strict-checks-in-missed-labels/2117
How is this change going?